hail/src/main/scala/is/hail/expr/ir/Emit.scala

```diff
@@ -877,7 +877,7 @@ class Emit[C](
           cb += srvb.advance()
         }, "aggs_result")
 
-        sc.store(cb)
+//        sc.store(cb)
```

Comment:
Why do we need to take this out? Store has logic in it that releases region references for aggregator state fields.

Reply:
ResultOP can be called multiple times. If it doesn't load, it can't store.

Reply:
Womp. Do we have a thing to clean up region references post-aggregation/can we guarantee that they're cleared if we take this out?

Reply:
I'll investigate this...

Reply:
Kinda think RunAgg should clean up after